docs/content/guides/dagster/branch_depl_with_snowflake.mdx

```diff
@@ -0,0 +1,325 @@
+
+With these tools, we can merge changes with confidence in the impact on our data platform and with assurance that our code will execute as intended.
+
+### Prerequisites
+
+<Note>
+  This guide is an extension of the{" "}
+  <a href="https://docs.dagster.io/guides/dagster/local-to-production-tutorial">
+    Transitioning Data Pipelines from Development to Production
+  </a>{" "}
+  guide, illustrating a workflow for staging deployments. We’ll use the examples
+  from this guide to build a workflow atop Dagster Cloud’s branch deployment
+  feature.
+</Note>
+
+This guide utilizes GitHub actions to trigger deployment—you’ll need to be using GitHub as your version control platform. You’ll also need to follow the [branch deployment setup guide](https://docs.dagster.cloud/guides/branch-deployments) and clone the [branch deployments quickstart template](https://github.com/dagster-io/dagster-cloud-branch-deployments-quickstart). After doing this, you should have:
+
+- A running agent to serve your branch deployments
+- A Dagster project set up for branch deployments containing the following:
+  - `branch_deployments.yml` GitHub workflow file
+  - Dagster repository (we’ve named ours `repo`)
+  - Dockerfile that installs your Dagster project
+
+If you’re new to Dagster, here’s an overview of the main concepts we’ll be using:
+
+- [Asset](https://docs.dagster.io/concepts/assets/software-defined-assets) - an asset is a software object that models a data asset. The prototypical example is a table in a database or a file in cloud storage.
+- [Resource](https://docs.dagster.io/concepts/resources) - a resource is an object that models a connection to a (typically) external service. Resources can be shared between assets and different implementations of resources can be swapped in depending on environment. For example, a resource may provide methods to send messages in Slack.
+- [IO manager](https://dagster-git-jamie-local-to-prod-example-elementl.vercel.app/concepts/io-management/io-managers) - an IO manager is a special kind of resource that handles storing and loading assets. For example, if we wanted to store assets in S3, we would use Dagster’s built in S3 IO manager.
+- [Op](https://docs.dagster.io/concepts/ops-jobs-graphs/ops) - an op is a unit of computation that accepts inputs and passes outputs to other ops.
+- [Graph](https://docs.dagster.io/concepts/ops-jobs-graphs/graphs) - a graph is an interconnected set of ops.
+- [Job](https://docs.dagster.io/concepts/ops-jobs-graphs/jobs#jobs) - a job is the main unit of execution in Dagster. Internally, it is a graph bound to a set of resources and execution configuration.
+
+### Overview
+
+We have a `PRODUCTION` Snowflake database with a schema named `HACKER_NEWS`. In our production cloud environment, we’d like to write tables to Snowflake containing subsets of Hacker News data. These tables will be:
+
+- `items` (the entire dataset)
+- `comments` (data about comments)
+- `stories` (data about stories)
```

Comment:
Why the parentheses instead of just a list with dashes? I find this formatting a bit odd. In my head, I read parentheses as an aside, like someone whispering to me or tapping my shoulder. 

```suggestion
- `items` - A table containing the entire dataset
- `comments` - A table containing data about comments
- `stories` - A table containing data about stories
```